### ISSUE TITLE
Non-existent password ref

### ISSUE THREAD

**Issue by rtibbles** [learningequality/kolibri#11769]
## Observed behavior
In the https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/plugins/setup_wizard/assets/src/views/SelectSuperAdminAccountForm.vue component, `this.$refs.password` is always undefined, which can give an unhandled error during certain operations.

## Errors and logs
First reported here: https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/plugins/setup_wizard/assets/src/views/SelectSuperAdminAccountForm.vue

## Expected behavior
References to non-existent refs should be removed.

## User-facing consequences
Seemingly none.

**Comment by himanshuc3** [comment]
@rtibbles It seems like most places have a check for it's access. If it's undefined everytime, where is it being used and why is it being referenced?

**Comment by rtibbles** [comment]
My guess is that this is the result of a copy paste of code, or changes in the component template that were not matched in the component script. We should just remove any that are actually undefined to avoid this.

**Comment by himanshuc3** [comment]
Can you help me with reproducing this error though. I can't see where the form is triggered. I feel the paths should have a clearer 1:1 mapping from the path string to the path user enters in browser.
- I tried directly hitting the path `import-facility/select-admin` but it gives a 404. 
- Going to Device/Facilities page and trying to import facility, I am not able to get past `Select Device` modal.
There needs to be a test facility that can be imported with dummy creds to test this flow.

**Comment by MisRob** [comment]
Hi @himanshuc3, I don't understand exactly this issue but from your last comment it sounds that you need to have access to the import facility feature? If that's the case, one way to achieve that is to run another Kolibri instance as described here https://kolibri-dev.readthedocs.io/en/develop/howtos/another_kolibri_instance.html

**Comment by himanshuc3** [comment]
thanks @MisRob let me try that and get back.

**Comment by himanshuc3** [comment]
`SelectSuperAdminForm` doesn't come in the flow that I am going through @MisRob .
![Screenshot from 2024-03-20 10-38-29](https://github.com/learningequality/kolibri/assets/19166045/5b037d9b-cfee-42ce-ab8c-c5f9169628ea)
Here I click on `import facility` cta and select device -> select learning facility -> enter admin creds. Nowhere here is the form `SelectSuperAdminForm` triggered.

**Comment by MisRob** [comment]
Ah this one is tricky to find @himanshuc3.

- First run another Kolibri instance via pex as described in the guide I referenced earlier. Complete the setup wizard on that instance.
- Then run your Kolibri development server with a different KOLIBRI_HOME folder so that you have a brand new instance and can access the setup wizard ` KOLIBRI_HOME=~/.kolibri-2 yarn run devserver-hot`

Then you can follow this recording:
![super admin](https://github.com/learningequality/kolibri/assets/13509191/524e3183-2d7b-490f-a2e3-96b89b81ad25)

**Comment by himanshuc3** [comment]
I was initially using the command `KOLIBRI_HOME="<foldername>" python <filename>.pex start`. It was working but somehow I still wasn't getting to the component in the flow. 
Running using `yarn run devserver` with a different kolibri env is easier but it throws the error:
```
watch] /home/himanshu/Desktop/projects/learningequality/kolibri/node_modules/webpack-dev-server/lib/Server.js:2344
[watch]         throw error;
[watch]         ^
[watch] 
[watch] Error: listen EADDRINUSE: address already in use 0.0.0.0:3000
[watch]     at Server.setupListenHandle [as _listen2] (node:net:1817:16)
[watch]     at listenInCluster (node:net:1865:12)
[watch]     at doListen (node:net:2014:7)
[watch]     at process.processTicksAndRejections (node:internal/process/task_queues:83:21) {
[watch]   code: 'EADDRINUSE',
[watch]   errno: -98,
[watch]   syscall: 'listen',
[watch]   address: '0.0.0.0',
[watch]   port: 3000
[watch] }
[watch] 
[watch] Node.js v18.19.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
I don't see any related key in `options.ini` to change it, so don't know whether it's configurable.

**Comment by bjester** [comment]
@himanshuc3 On your second instance, instead of running the `yarn run devserver`, you could run:
1. `yarn run build` to build the frontend resources
2. `pip install -e .` to 'install' the kolibri package
3. `KOLIBRI_HOME="<foldername>" kolibri start --foreground --port <alt>` to start on `<alt>` port that you specify, with alternative `KOLIBRI_HOME`

**Comment by himanshuc3** [comment]
I am able to run the 2 instances now.  Thanks @bjester @MisRob for the help. 

There might be an unrelated bug though, because as soon as I try to use the earlier instance  (running on port 8000) which was ran using `yarn run devserver`  and go to `device -> facilities`  on it,page it logs me out instantaneously.

![Screenshot from 2024-03-27 13-42-46](https://github.com/learningequality/kolibri/assets/19166045/11723c34-315c-4c09-8419-65298235f27e)

**Comment by rtibbles** [comment]
Yes - unfortunately, cookies are shared across the same host, even if the port is different. So if you need to interact with both servers, you would need to use two separate browser sessions.

**Comment by jredrejo** [comment]
Fixed in #12077
